bin/base.sh

```diff
@@ -16,3 +16,13 @@ if ( command -v pigz >/dev/null 2>&1 ); then
 else
     export GZIP_CMD=gzip
 fi
+
+if ( command -v pv >/dev/null 2>&1 ); then
```

Comment:
What if we also add to this condition, only do `pv` if we are on a terminal?

Reply:
Done